### DISCUSSION TITLE
Why write modus

### DISCUSSION THREAD

**Discussion by tblock-zz** [quantbelt/ib_fundamental#14] [Q&A]
Hello,
thanks for this nice project.
I tried it out today after installing it with 
pip install ib-fundamental

I made a first test and get the error:
Error 321, reqId -1: Bei der Validierung der Anfrage ist ein Fehler aufgetreten.-'cr' : cause - Die API-Schnittstelle befindet sich derzeit im Read-Only-Modus. 
Which means it seems to need write acces.
So why is write access needed to the API for retrieving the fundamental data?
I want to avoid givin write acces to prevent unwanted actions.

Thanks for any help
Thomas

**Comment by gnzsnz** [comment]
I would need that you provide more details. Based on my quick test it works with API in read only mode

![image](https://github.com/user-attachments/assets/52d088ae-c0fe-4fea-85d2-d3d13f20e4fc)
